### DISCUSSION TITLE
Monorepo example using PNPM

### DISCUSSION THREAD

**Discussion by andrewdoro** [tamagui/tamagui#50] [Ideas]
I would love to see an example with PNPM.

**Comment by natew** [comment]
pnpm should work nearly identical to yarn beyond switching it out. The new create-tamagui-app starter repo should be easy to convert over.

**Comment by jarrodpayne** [comment]
`npm create tamagui --use-pnpm` appears to be broken. I get errors with `create-tamagui@1.2.2`. Even if this was working as expected, I suspect there are several issues with react-native and the way pnpm installs deps by default. 🤷🏻
